packages/web/server/api/edge-routers/authenticators.ts

```diff
@@ -0,0 +1,46 @@
+import { z } from "zod";
+
+import { createTRPCRouter, publicProcedure } from "~/server/api/trpc";
+import {
+  Authenticator,
+  queryAuthenticators,
+} from "~/server/queries/authenticators";
+import { queryCosmosAccount } from "~/server/queries/cosmos/auth";
+
+interface NestedAuthenticator {
+  authenticator_type: Authenticator["type"];
+  data: Authenticator["data"];
+}
+
+export const authenticatorsRouter = createTRPCRouter({
+  getAuthenticators: publicProcedure
+    .input(z.object({ address: z.string() }))
```

Comment:
If this must be the user's OSMO address, would be nice to have param name be more specific and have it be `userOsmoAddress` as we have with assets procedures